src/documents/TextToSpeechAPI.md

```diff
@@ -0,0 +1,583 @@
+---
+title: Text To Speech API
+layout: default
+---
+
+# Text To Speech API #
+
+The **Text To Speech** component uses [Web Speech Synthesis API](https://dvcs.w3.org/hg/speech-api/raw-file/tip/speechapi.html#tts-section) to queue up and read texts.
+
+## Browser Support ##
+The Text To Speech component can be used in browsers that support [Web Speech Synthesis API](https://dvcs.w3.org/hg/speech-api/raw-file/tip/speechapi.html#tts-section) is supported. At the moment when this document is written, Mar 4 2015, these browsers include:
```

Comment:
The "is supported" at the end of the first sentence is unnecessary as you've already mentioned "that support" earlier.

For the second sentence you may want to phrase it as "At the time of writing, Mar 4 2015, these browsers include:" and maybe include a link to [caniuse](http://caniuse.com/#feat=web-speech)